js/carousel.js

```diff
@@ -0,0 +1,33 @@
+const delay = 5000; //ms
+const slides = document.querySelector(".slides");
+const slidesCount = slides.childElementCount;
+const maxLeft = (slidesCount - 1) * 100 * -1;
+
+let current = 0;
+
+function changeSlide(next = true) {
+  if (next) {
+    current += current > maxLeft ? -100 : current * -1;
```

Comment:
Nitpick: `current` and `maxLeft` could have a bit more expressive names for easier understand of their intent.